# Yaml file with configutration of sensored setup
#
# markerFixList: cointains the label of the
# markers that are fixed on the setup
# markerFixPos_##: contains values that are
# a geometry_msgs::Pose: - x, y, z, qx, qy, qz, qw

# for example:
# markerFixPos_23: [0.1, 0.1, 0.0, 0.0, 0.0, 0.0, 1.0]
# where the fix marker 23 is located at position
# x, y, z =>  0.1, 0.1 , 0.0 in meters
# with the rotation
# x, y, z, q => 0.0, 0.0, 0.0, 1.0

# camera_frame: contains the string that define
# the camera frame wrt the markers are referred

tablecalibrator:
  ros__parameters:
    markerFixList: [90, 92, 93, 91]
    markerFixPos_90: [0.04, 0.04, 0.0, 0.0, 0.0, 0.0, 1.0]
    markerFixPos_92: [0.54, 0.39, 0.0, 0.0, 0.0, 0.0, 1.0]
    markerFixPos_93: [0.54, 0.04, 0.0, 0.0, 0.0, 0.0, 1.0]
    markerFixPos_91: [0.04, 0.39, 0.0, 0.0, 0.0, 0.0, 1.0]
    camera_frame: usb_ms_cam   # camera frame wrt the markers are referred
  
  